lib/doorkeeper/config.rb

```diff
@@ -241,6 +241,11 @@ def option(name, options = {})
     option :grant_flows,                    default: %w[authorization_code client_credentials]
     option :handle_auth_errors,             default: :render
 
+    # Allow optional hashing of input tokens to some other representation
+    # that is persisted. Can be used for hashing of input token,
+    # grants and application secret values.
+    option :secret_hashing, default: nil
```

Comment:
I prefer (and think it's something like an enterprise standard)  `encrypt_secrets` instead of `hashing`, WDYT?

Reply:
It's only terminology and I will accept your decision, but for me `encrypt_secrets` bears no resemblance to what this option will do. It will irreversably map all plain text tokens with a one-way function, so encryption does not really fit the option in my opinion.

Reply:
OK, now I've get the idea. Let's rename it to `hash_secrets`, WDYT (like action, `reuse token`, `enable_application_owner` , `enforce_content_type`, etc)?

Reply:
Aye! :+1: